.github/workflows/test.yml

```diff
@@ -33,7 +33,18 @@ jobs:
         run: |
           nix-env -iA cachix -f https://cachix.org/api/v1/install
           cachix use holochain-ci
-      - name: run-tests
+
+      - name: Prepare Nix environment
         run: |
           cd $GITHUB_WORKSPACE
-          nix develop --command bash -c "pnpm i && pnpm test"
\ No newline at end of file
+          nix develop --command bash -c "pnpm i && echo Completed"
+
+      - name: debug-tsc-version
+        run: |
+          pushd tests/
+          pnpm ls typescript
```

Comment:
This won't ever work. You need to wrap it with the same `nix develop --command bash -c "pnpm ls typescript"` as the rest of them.